internal/cephfs/store/fsjournal.go

```diff
@@ -128,6 +128,14 @@ func CheckVolExists(ctx context.Context,
 		}
 		err = cloneState.ToError()
 		if errors.Is(err, cerrors.ErrCloneInProgress) {
+			progressReport := cloneState.GetProgressReport()
+			err = fmt.Errorf("%w. progress report: percentage cloned=%s, amount cloned=%s, files cloned=%s",
```

Comment:
>progress report: percentage cloned=, amount cloned=, files cloned= 

We might need to put it behind some if-check to avoid this kind of log in unsupported version(it looks ugly).

Reply:
Added a check for it.